lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -831,6 +843,21 @@ def config_apn_correction(self, cmd):
         else:
             print("APN Correction failed")
 
+    def config_health_service(self, cmd: health_service_cmds):
+        """
+        Configure magma@health service on access gateway
+        Args:
+            cmd: Enable / Disable cmd to configure service
+        """
+        magma_health_service_name = "health"
+        if cmd.name == MagmadUtil.health_service_cmds.DISABLE.name:
+            if self.is_service_enabled(magma_health_service_name):
+                self.disable_service(magma_health_service_name)
+            print("Health service is disabled")
+        elif cmd.name == MagmadUtil.health_service_cmds.ENABLE.name:
+            self.enable_service("health")
```

Comment:
Does it make sense to check `is_service_enabled` here as well, and only enable if the call returns false?